v2/tools/generator/internal/codegen/code_generator.go

```diff
@@ -82,50 +82,56 @@ func NewCodeGeneratorFromConfig(configuration *config.Configuration, idFactory a
 func createAllPipelineStages(idFactory astmodel.IdentifierFactory, configuration *config.Configuration) []pipeline.Stage {
 	return []pipeline.Stage{
 
-		pipeline.LoadSchemaIntoTypes(idFactory, configuration, pipeline.DefaultSchemaLoader),
-
-		// Import status info from Swagger:
-		pipeline.AddStatusFromSwagger(idFactory, configuration),
+		// Import data from Swagger:
+		pipeline.LoadTypesFromSwagger(idFactory, configuration),
+		pipeline.SimplifySwaggerNames(idFactory, configuration),
+		// De-pluralize resource types
+		// (Must come after type aliases are resolved)
+		// pipeline.ImproveResourcePluralization(),
 
 		// Reduces oneOf/allOf types from schemas to object types:
 		pipeline.ConvertAllOfAndOneOfToObjects(idFactory),
+		pipeline.StripUnreferencedTypeDefinitions(),
 
 		// Flatten out any nested resources created by allOf, etc. we want to do this before naming types or things
 		// get named with names like Resource_Spec_Spec_Spec:
 		pipeline.FlattenResources(),
 
 		// Copy additional swagger-derived information from status into spec
-		pipeline.AugmentSpecWithStatus().RequiresPrerequisiteStages("allof-anyof-objects", "addStatusFromSwagger"),
+		// pipeline.AugmentSpecWithStatus().RequiresPrerequisiteStages("allof-anyof-objects", "addStatusFromSwagger"),
```

Comment:
Do we need to keep this commented out code? If so, move the pre-requisite into the stage definition.